src/ui/osx/TogglDesktop/test2/TogglDesktop-Prefix.pch

```diff
@@ -5,3 +5,5 @@
 #ifdef __OBJC__
     #import <Cocoa/Cocoa.h>
 #endif
+
+#import "toggl_api.h"
```

Comment:
By adding `toggl_api.h` in pre-complied header, it's optimized the build-time and reduce the redundant of import statement